sdk/keyvault/azure-security-keyvault-keys/sample/get-key/main.cpp

```diff
@@ -38,9 +38,6 @@ using namespace Azure::Security::KeyVault::Keys;
 
 int main()
 {
-  Azure::Core::Logging::SetLogListener(
```

Comment:
BTW, now you can set `AZURE_LOG_LEVEL` to `verbose` (/`informational`/`warning`/`error`) in your environment, and it will be printing!